azure_media_services/static/js/player.js

```diff
@@ -155,7 +164,8 @@ function initTranscript(player, transcript, transcriptElement) {
   transcriptElement.append(html);
 
   // handle events when user clicks on transcripts
-  $('.azure-media-xblock-transcript-element').click(function(evt) {
+  $('.azure-media-xblock-transcript-element').on('click keypress',function(evt) {
```

Comment:
Shouldn't you filter out which key is pressed here? Similar to toggling the transcript? (i.e. only change transcript element if spacebar or enter is pressed, not any key) Not sure but let me know.

Also remove this console.log